### PR TITLE
Implement initial version of credentials/v2 context.

### PR DESCRIPTION
**Pull request by msporny**
This PR creates the initial version of the credentials/v2 context, which continues to be a work in progress, and unstable, in the W3C VCWG.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    "rollup-plugin-commonjs": "^10.1.0",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.12"
+  },
+  "engines": {
+    "node": ">=16"
+  },
+  "scripts": {
+    "build": "npm run clean && npm run serialize && webpack && rollup -c",
+    "clean": "rm -rf dist/* && rm -rf contexts/*.jsonld",
```
Comment (msporny) [inline package.json:61]: This is weird, why are we doing this? Why are we checking in a file and then auto-generating it on build? Is this something that's used when just doing a `git clone` w/o a build?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    "lint": "eslint js test",
+    "prepare": "npm run build",
+    "serialize": "node bin/serialize.js",
```
Comment (msporny) [inline package.json:64]: This should probably be a tool that's externalized?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    "url": "https://github.com/digitalbazaar/credentials-v2-context/issues"
+  },
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com/"
+  },
+  "homepage": "https://github.com/digitalbazaar/credentials-v2-context#readme",
+  "license": "BSD-3-Clause",
+  "devDependencies": {
```
Comment (msporny) [inline package.json:39]: 102MB of dependencies... for a JSON-LD context file. This is the world that webpack/rollup/esm has given us.

Comment (aljones15) [reply package.json:39]: Dev Deps are not in the production build provided that is where the 102 MB comment comes from. Only the base deps are in the package npm will serve.

File: package.json (hunk 2 above)
Comment (aljones15) [reply package.json:64]: You can register this as an npm binary and use `npx serialize`. That is done by adding a `bin` section to the `package.json`.

File: package.json (hunk 1 above)
Comment (aljones15) [reply package.json:61]: It looks like this removes the dist dir (just to be extra-sure) and then generates a new file in the dist dir.

**Review by davidlehn — APPROVED** [review]
Merging to main to have a base that will track the latest spec context.

**Pull request merged**